_posts/2015-12-7-greenkeeper.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Greenkeeper"
+author: Jan
+layout: blog-post-tgif-ish
+comments: true
+---
+
+This is Part 3 of of our [little series about what has happened in Hoodie-land](http://hood.ie/blog/we-have-been-busy.html "We. Have. Been. Busy!") in the past <strike>three</strike>five months.
+
+Stop worrying about npm dependencies breaking your build: Greenkeeper saves you time and money by automating dependency updates and notifying you of failed builds so you can spend your development time actually making things.
```

Comment:
Might be nice to have a picture of the greenkeeper logo somewhere around this paragraph? :)